arangod/Aql/Functions.cpp

```diff
@@ -1313,6 +1313,13 @@ Result parseShape(ExpressionContext* exprCtx,
   }
 }
 
+std::string getFunctionName(const AstNode& node) {
+
```

Comment:
add `TRI_ASSERT` to ensure that the specified `AstNode` is of type `NODE_TYPE_FCALL`